charts/federation-v2/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.0.2
+appVersion: 0.0.7
```

Comment:
This entire commit needs to be removed.  

 - the chart version should not be updated in the tree.  It should only be set in the archived chart.
 - checking large binary files into git is a no-no.  It inflates the size of the tree unnecessarily.
 - The index will need to be created as part of the release process.

Reply:
Removing.